### PR TITLE
#358: Update html_header.php

### PR DESCRIPTION
**Pull request by brittainmark**
Have updated html header to font Awesome 6
Corrected error that stopped preload working
change loads to loop rather than hard coded list

Have left in the fa4-shim as bootstrap was on fa-5 it is probably not required. will take out if you don't want it.

### CODE REVIEW

**Comment by lat9** [comment]
I'm not ready to deal with this for the current, almost ready, release.  I'll hold off any additional thoughts until the template's v3.7.0 is underway.

**Comment by brittainmark** [comment]
Had missed out the no preload. Did it now so I did not forget.

File: includes/extra_datafiles/dist.site-specific-bootstrap-settings.php [hunk 1]
```diff
@@ -23,3 +23,8 @@
 // (true) to disable the back-to-top control on each page.
 //
 //$zca_disable_back_to_top = false;
+
+// ----
+// This controls preloading of fontawesome, jquery and boostrap css
+// if you don't want to preload the scrips and css uncomment line below
+//$bs4_no_preloading = true;
```
Comment (lat9) [inline includes/extra_datafiles/dist.site-specific-bootstrap-settings.php:30]: Let's keep with the current naming convention for variables here, use `$zca_no_preloading` and (like the other soft-settings) the value present here is the default (`false`).

File: includes/templates/bootstrap/common/html_header.php [hunk 2]
```diff
@@ -2,7 +2,7 @@
 /**
  * Common Template
  *
- * BOOTSTRAP v3.5.0
+ * BOOTSTRAP v3.6.0
```
Comment (lat9) [inline includes/templates/bootstrap/common/html_header.php:5]: This will be for `3.7.0`.

**Review by lat9 — CHANGES_REQUESTED** [review]
You'll also need to 'rebase' on the current default branch.

**Comment by dennisns7d** [comment]
Noting that the latest version of FA is 6.5.2.

**Comment by lat9** [comment]
> Noting that the latest version of FA is 6.5.2.

Good point, what about that, @brittainmark?

**Comment by lat9** [comment]
Also, please re-rebase on the v370 branch; that's where this change is destined.

**Review by lat9 — APPROVED** [review]
Thanks, @brittainmark, for these and other changes.

**Pull request merged**